examples/basics/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "dev": "astro dev",
-    "start": "astro dev",
+    "start": "echo 'Warning: The `start` script is no longer included in Astro templates. Please use `dev` to start the development server.'",
```

Comment:
I'm in favor of just getting rid of these. Having warnings in *starter* templates feels a bit odd to me.

Reply:
Sounds good. I stacked commits, so I can just revert the second one